content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
+date: 2022-04-28T12:00:00+0000
```

Comment:
We'll need to choose what day this will go out -- just commenting so that we don't forget if it gets merged in before release day.

Reply:
@nate-double-u I think the ask was to get this out for public visibility sooner rather than later? Or must this wait for GA to go live?

Reply:
I'm happy for it to go out before release day too.
/cc @JamesLaverack @mickeyboxell